### ISSUE TITLE
Some site content not accessible at 200% would fail WCAG 1.4.4

### ISSUE THREAD

**Issue by davidpmccormick** [wellcomecollection/wellcomecollection.org#12358]
### What

This was unearthed as part of #12350 but probably not exclusive to that page. The problem seems to stem from an element having `white-space: nowrap`

### Where

Seen on /collections

<img width="530" height="982" alt="Image" src="https://github.com/user-attachments/assets/51addbf4-16fe-46c2-aa70-26688acdb6df" />
